code/modules/shuttle/super_cruise/orbital_map_components/orbital_objects/star.dm

```diff
@@ -0,0 +1,20 @@
+
+/datum/orbital_object/star
+	name = "Stellar Body"
+	mass = 100000
+	radius = 200
+	static_object = TRUE
+
+/datum/orbital_object/star/collision(datum/orbital_object/other)
+	//You got lucky this time
+	if(other.collision_ignored || collision_ignored)
+		return
+	//You didnt get lucky this time
+	message_admins("Orbital body [other.name] collided with the star [name] and was destroyed.")
+	log_game("Orbital body [other.name] collided with the star [name] and was destroyed.")
+	//Rip.
+	if(istype(other, /datum/orbital_object/star))
+		//Make a black hole or something
```

Comment:
OwO

Reply:
Stars were removed due to the fact that attempting to orbit the station while its orbiting lavaland orbiting a star was a near impossible feat

Reply:
sad

Reply:
how can you orbit a station

Reply:
i mean, you can orbit ANYTHING. it does raise the question of how close orbiting a man-made station with a man-made spaceship would be to a binary system